roles/edpm_ovn/defaults/main.yml

```diff
@@ -90,6 +88,7 @@ edpm_ovn_ovs_external_ids:
 edpm_ovn_ovs_other_config:
   vlan-limit: 0
 
+# TODO: clean these up even more?
```

Comment:
I would avoid open ended TODOs like this one. Granted it may be an option in the future,  but if there is really something to refactor we should make it into a proper task, this way it's just going to get forgotten, until someone else comes along and decides to do it without context.